components/BottomAppBar/README.md

```diff
@@ -115,3 +119,62 @@ tracking its development:
 - Task: [Implement a color themer](https://www.pivotaltracker.com/story/show/157095394)
 - Task: [Impement a surface variant themer](https://www.pivotaltracker.com/story/show/156933141)
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+To help ensure your bottom app bar is accessible to as many users as possible, please be sure to review the
+following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate
+[`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel)
+to all buttons within the bottom app bar.
+
+#### Swift
+
+```swift
+bottomAppBar.floatingButton.accessibilityLabel = "Compose"
+```
```

Comment:
Are there any other button apis on the bottom app bar that need to be set?

Reply:
Done